main.py

```
@@ -0,0 +1,581 @@
+        if data == 'referral_program':
+            await show_referral_program(query, context, lang, user_id)
+            return
+
+        # Добавьте другие обработчики кнопок по аналогии...
+
+    except Exception as e:
+        logger.error(f"Error in button callback: {e}", exc_info=True)
+        try:
+            await query.message.reply_text("Произошла ошибка.", parse_mode='HTML')
+        except:
+            pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)